js/map.js

```diff
@@ -310,14 +310,15 @@ function customMarkerGenerateJSON() {
 
   var currentJSONDate = (new Date()).toJSON();
 
-  var sJSON = '<div class="box-header"><h2>JSON for node</h2></div><pre style="white-space: pre;margin-bottom:10px;">{\n' +
-    '  "name": "Unnamed Node",\n' +
-    '  "latitude": ' + lat + ',\n' +
-    '  "longitude":' + lng + ',\n' +
-    '  "cardinalDirection": "' + dir + '",\n' +
-    '  "floor": ' + floor + ',\n' +
-    '  "dateAdded": "' + currentJSONDate + '"\n' +
-    '}\n</pre>';
+  var sJSON = '<div class="box-header"><h2>JSON for node</h2></div><pre style="white-space: pre;margin-bottom:10px;">   {\n' +
```

Comment:
Extra spaces after the start of `<pre>`.

Reply:
By Design

To Align with the existing nodelist format indent the first { 3 spaces

Reply:
The 3 spaces in the node list is very strange. I think everywhere else I've seen, it's either 2 or 4 spaces indents.

Reply:
We would need to change the node list them
It was generated  by default settings using https://jsonformatter.curiousconcept.com/
And its defaulted to 3...

Reply:
If it's only me complaining let's keep it as is, maybe I just have no seen enough JSON :)